package.json

```diff
@@ -32,7 +32,7 @@
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.4.2",
+    "@ember/test-helpers": "2.7.0",
```

Comment:
This is needed because the 3.4 LTS scenario fails due to `@ember/test-helpers` 2.8 including a peerDependency on ember-source 3.8. ember-try doesn't seem to have a way to add the `--force` argument to the npm install command for a specific scenario, but I might be wrong.

I can split this of in a separate PR if needed, but wanted to have the commit here so CI could be run properly.

Reply:
I fixed this in #46 👍